src/handlers/notification/index.js

```diff
@@ -150,120 +149,127 @@ const processMessage = async (msg) => {
     }
 
-    if (actionLower === 'commit' && statusLower !== 'success') {
-      let callbackURL = await Participant.getEndpoint(from, FSPIOP_CALLBACK_URL_TRANSFER_ERROR, id)
-      return Callback.sendCallback(callbackURL, 'put', content.headers, content.payload, id, from)
+    if (actionLower === ENUM.transferEventAction.COMMIT && statusLower !== ENUM.messageStatus.SUCCESS) {
+      let callbackURLFrom = await Participant.getEndpoint(from, FSPIOP_CALLBACK_URL_TRANSFER_ERROR, id)
+
+      let methodFrom = ENUM.methods.FSPIOP_CALLBACK_URL_TRANSFER_ERROR
+
+      Logger.debug(`Notification::processMessage - Callback.sendCallback(${callbackURLFrom}, ${methodFrom}, ${JSON.stringify(content.headers)}, ${JSON.stringify(content.payload)}, ${id}, ${from}, ${to})`)
+      return Callback.sendCallback(callbackURLFrom, 'put', content.headers, content.payload, id, from, to)
     }
 
-    if (actionLower === 'reject') {
+    if (actionLower === ENUM.transferEventAction.REJECT) {
       let callbackURLFrom = await Participant.getEndpoint(from, FSPIOP_CALLBACK_URL_TRANSFER_PUT, id)
       let callbackURLTo = await Participant.getEndpoint(to, FSPIOP_CALLBACK_URL_TRANSFER_PUT, id)
 
-      // send an extra notification back to the original sender.
-      // first remove any JWS related headers so we dont bounce them back.
-      // also set source to "switch" to make it clear this notification originates at the switch
-      headers = removeJwsHeaders(Object.assign({}, content.headers, { 'FSPIOP-Destination': from }, { 'FSPIOP-Source': 'switch' }))
-      await Callback.sendCallback(callbackURLFrom, 'put', headers, content.payload, id, from)
+      let methodFrom = ENUM.methods.FSPIOP_CALLBACK_URL_TRANSFER_PUT
+      let methodTo = ENUM.methods.FSPIOP_CALLBACK_URL_TRANSFER_PUT
 
       // forward the reject to the destination
-      headers = Object.assign({}, content.headers, { 'FSPIOP-Destination': to }, { 'FSPIOP-Source': from })
-      return Callback.sendCallback(callbackURLTo, 'put', headers, content.payload, id, to)
+      Logger.debug(`Notification::processMessage - Callback.sendCallback(${callbackURLTo}, ${methodTo}, ${JSON.stringify(content.headers)}, ${JSON.stringify(content.payload)}, ${id}, ${from}, ${to})`)
+      await Callback.sendCallback(callbackURLTo, methodTo, content.headers, content.payload, id, from, to)
+
+      // send an extra notification back to the original sender.
+      Logger.debug(`Notification::processMessage - Callback.sendCallback(${callbackURLFrom}, ${methodTo}, ${JSON.stringify(content.headers)}, ${JSON.stringify(content.payload)}, ${id}, ${ENUM.headers.FSPIOP.SWITCH.value}, ${from})`)
+      return Callback.sendCallback(callbackURLFrom, methodFrom, content.headers, content.payload, id, ENUM.headers.FSPIOP.SWITCH.value, from)
     }
 
-    if (actionLower === 'abort') {
+    if (actionLower === ENUM.transferEventAction.ABORT) {
```

Comment:
Here is another example, which would need .toLowerCase() if we keep the top one, but for me this is more appropriate.